### ISSUE TITLE
Employee Department MYSQL Creation Issue

### ISSUE THREAD

**Issue by ashishsme14** [sfu-db/connector-x#83]
We had given task as below attachment -

![Employee Issue 1](https://user-images.githubusercontent.com/61864865/122409848-18577080-cfa1-11eb-897b-2e6f7fb14495.JPG)

We written below query which is not working i.e. it does not fetching desire result -

```
With New Dept_id as (select  x.dpt_code from department x where x.dpt_code not in 
(select e.dpt_id from emplyee e join
employee e where e.dep_id=d.dpt_code))
With New Dept_name as (select  x.dpt_name from department x )
SELECT employee.emp_id, employee.emp_name, employee.hire_date, employee.jon_name, employee.dept_id, New Dept_id, New Dept_name from employee JOIN department on (employee.dep_id=department.dpt_code) JOIN 
```

**Comment by wangxiaoying** [comment]
Thank you for the report. Currently we only support read sql from Postgres, but MySQL is on the way.

**Comment by dovahcrow** [comment]
Hi @ashishsme14 we just released connectorx 0.2.0a2 version, which includes MySQL and sqlite support. You can try it using
`pip install -U connectorx==0.2.0a2`.
